sparkler-core/sparkler-ui/pom.xml

```diff
@@ -46,6 +46,7 @@
     </dependencies>
 
     <build>
+        <finalName>${project.artifactId}</finalName>
```

Comment:
so we don't include a version id on the artifact. Allows us to symlink to a constant path when setting up the docker dev env.